### ISSUE TITLE
Offer submitting (reviewed) word list to language pack maintainer

### ISSUE THREAD

**Issue by PanderMusubi** [AnySoftKeyboard/AnySoftKeyboard#1071]
In order to enhance the language packs, could each personal word list please get a function that users can send their word list per email to the maintainer of the language pack?

**Comment by PanderMusubi** [comment]
Perhaps a warning with cancel option is in place to remind people to review their list so it doesn't contain any sensitive data such as passwords.

**Comment by menny** [comment]
that's an interesting idea. Let me think about this for a while. 

One thing: I would prefer a way for users to submit their word-list to central, open, space where anyone can use it.

**Comment by PanderMusubi** [comment]
Also fine with me. Store the feedback under the namespace of the language pack. Special GitHub repo where automatically an issue gets created or as a git commit under a UUID for each feedback?

**Comment by PanderMusubi** [comment]
Any more ideas on this? Contributions via personal word lists do need to be looked at by a maintainer. Just collecting them and making them available would open the door for a lot of common spelling mistakes.

As Android has a lot of share functionality, I would prefer to offer a function to share your personal word list with only the maintainer of that list via an email (via the email application the user has installed).

**Comment by menny** [comment]
So, each language-pack will have a maintainer? And their email address will be published?

**Comment by PanderMusubi** [comment]
Only if they provide one, I'd say. We have a generic one for the Dutch word list.

**Comment by menny** [comment]
I'm trying to visualize the process:
The dictionary entry will have a support email (as it has a name, description, etc today).
In AnySoftKeyboard there will be a new button in the languages settings to share the user-word list via email to that email address.

The maintainer will go over those lists and add them to language pack (after removing any typos).

Is that what you had in mind?

**Comment by PanderMusubi** [comment]
Yes. The email does not have to be visible in the app, only when you use that button it will be used in creating a draft email. The app NewPipe has something like this to report URLs that result in errors.

**Comment by PanderMusubi** [comment]
Can you remove the Stale labels from some issues, otherwise they get closed automatically. Thanks
